Code/GraphMol/MarvinParse/MarvinDefs.h

```diff
@@ -252,20 +306,40 @@ class MarvinMolBase {
       ,
       int &bondCount  // starting and ending bond count
       ,
-      int &sgCount);  // starting and ending sq count
-
- private:
-  void cleanUpNumberingMolsAtomsBonds(
-      int &molCount  // this is the starting mol count, and receives the ending
-                     // mol count - THis is used when
-                     // MarvinMol->convertToSuperAtaoms is called multiple times
-                     // from a RXN
+      int &sgCount  // starting and ending sq count
       ,
-      int &atomCount  // starting and ending atom count
+      std::map<std::string, std::string>
+          &sgMap  // map from old sg number to new sg number
+      ,
+      std::map<std::string, std::string>
+          &atomMap  // map from old atom number to new atom number
```

Comment:
If they are numbers why not use numbers?

Reply:
As you probably guessed, they are not numbers.  Most are like a13 a14 etc,  but while converting some get thinks like
a13_sg2_3 for uniqueness.